settings.gradle

```diff
@@ -0,0 +1,65 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ *
+ * The settings file is used to specify which projects to include in your build.
+ *
+ * Detailed information about configuring a multi-project build in Gradle can be found
+ * in the user manual at https://docs.gradle.org/8.0.2/userguide/multi_project_builds.html
+ */
+import java.nio.file.Paths
+
+plugins {
+    id("com.gradle.enterprise") version("3.16.2")
+    id 'org.gradle.toolchains.foojay-resolver-convention' version '0.7.0'
+}
+
+void addSubProjects(String path, File dir) {
+    if (dir.isDirectory() == false) {
+        return;
+    }
+    if (dir.name == 'buildSrc') {
+        return;
+    }
+    if (new File(dir, 'build.gradle').exists() == false) {
+        return;
+    }
+    if (findProject(dir) != null) {
+        return;
+    }
+
+    final String projectName = "${path}:${dir.name}"
+    include projectName
+
+    project(projectName).projectDir = dir
+
+    for (File subdir : dir.listFiles()) {
+        addSubProjects(projectName, subdir)
+    }
+}
+
+rootProject.name = 'MigrationsAssistant'
+include 'TrafficCapture:captureKafkaOffloader'
```

Comment:
Can we programmatically include these projects rather than list them?

Reply:
How are you thinking that they could be programmatically included?

Reply:
Let me rephrase - do we have to enumerate each project in this way? Seems like we have a method to find all projects with `addSubProjects()` that could be used.

Reply:
This might be a good idea, but I have enough changes in this PR.  We can revisit this behavior change later.